### PR TITLE
Bug 1733454: Specify kind in op registry labels

### PR DESCRIPTION
**Pull request by galletti94**
Every operator registry pod is created with a label
"marketplace.catalogsourceconfig" even though it may be created by an
OperatorSource. This fixes this issue by gating on the kind before
assigning these labels to the operator registry pod.

### CODE REVIEW

File: pkg/registry/registry.go [hunk 1]
```diff
@@ -250,8 +250,15 @@ func (r *registry) ensureServiceAccount() error {
 
 // getLabels returns the label that must match between the Deployment's
```
Comment (aravindhp) [inline pkg/registry/registry.go]: Nit: Given you are in the area please change to `getLabel`.

**Comment by aravindhp** [comment]
/hold

Please open a BZ for this as this cannot be merged without a BZ reference.

**Comment by galletti94** [comment]
@aravindhp I addressed your comment. PTAL when you get a chance

**Comment by aravindhp** [comment]
/hold cancel

@galletti94 lgtm

**Pull request merged**